### DISCUSSION TITLE
JBoss Logging Micrometer instrumentation

### DISCUSSION THREAD

**Discussion by AndreasPetersen** [quarkusio/quarkus#37579] [Q&A] [answered]
According to [Quarkus' Micrometer documentation](https://quarkus.io/guides/telemetry-micrometer#quarkus-extensions-using-micrometer) [jboss-logging](https://quarkus.io/guides/logging) is instrumented with Micrometer. However, on the documentation for logging, I can find no mention of it. Likewise, I can find no binder configuration property for logging. Looking at the produced metrics by Quarkus, there are also no metrics about logs.

How do I activate logging metrics?

[a comment by an automated account is not shown]

**Comment by ebullient** [comment, marked as answer]
`quarkus.log.metrics.enabled=true`

It is present here: https://quarkus.io/guides/all-config, but should be present on the logging page.

Fixed by PR https://github.com/quarkusio/quarkus/pull/37639

**Reply by AndreasPetersen** [reply]
Amazing, thanks @ebullient
